pkg/types/organization.go

```diff
@@ -1,17 +1,34 @@
 package types
 
 import (
+	"context"
+	"time"
+
+	"github.com/SigNoz/signoz/pkg/valuer"
 	"github.com/uptrace/bun"
 )
 
-// TODO: check constraints are not working
 type Organization struct {
 	bun.BaseModel `bun:"table:organizations"`
 	TimeAuditable
-	ID              string `bun:"id,pk,type:text" json:"id"`
-	Name            string `bun:"name,type:text,notnull" json:"name"`
-	IsAnonymous     bool   `bun:"is_anonymous,notnull,default:0,CHECK(is_anonymous IN (0,1))" json:"isAnonymous"`
-	HasOptedUpdates bool   `bun:"has_opted_updates,notnull,default:1,CHECK(has_opted_updates IN (0,1))" json:"hasOptedUpdates"`
+	Identifiable
+	Name        string `bun:"name,type:text,nullzero" json:"name"`
+	Alias       string `bun:"alias,type:text,nullzero" json:"alias"`
+	DisplayName string `bun:"display_name,type:text,notnull" json:"displayName"`
+}
+
+func NewDefaultOrganization(displayName string) *Organization {
```

Comment:
```suggestion
func NewOrganization(displayName string) *Organization {
```

We will create a separate function for Default which won't take name or displayName in input

Reply:
duneee